rackcity/utils/rackcity_utils.py

```diff
@@ -58,6 +58,8 @@ def validate_asset_location_in_rack(
     change_plan=None,
     related_asset_id=None,
 ):
+    if asset_rack_position is None:
+        return
     new_asset_location_range = [asset_rack_position + i for i in range(asset_height)]
```

Comment:
THIS LIST COMPREHENSION WAS THROWING AN EXCEPTION DEEP IN THE DEPTHS OF ASSET SAVING